doc/api/fs.md

```diff
@@ -3788,7 +3798,7 @@ If the third argument is a number, then it specifies `flags`, as shown in the
 following example.
 
 ```js
-const fs = require('fs');
+const fsPromises = require('fs').promises;
 const { COPYFILE_EXCL } = fs.constants;
```

Comment:
fs is undefined

Reply:
I'll fix it.

Reply:
Fixed. https://github.com/nodejs/node/pull/20838/commits/726be355b732e52545cd79da21eef0fe3f4d61fb